client/state/reader/lists/test/actions.js

```diff
@@ -63,53 +54,25 @@ describe( 'actions', () => {
 	} );
 
 	describe( '#followList()', () => {
-		useNock( ( nock ) => {
-			nock( 'https://public-api.wordpress.com:443' )
-				.post( '/rest/v1.2/read/lists/restapitests/testlist/follow' )
-				.reply( 200, {
-					following: true,
-				} );
-		} );
-
-		test( 'should dispatch fetch action when thunk triggered', () => {
-			followList( 'restapitests', 'testlist' )( spy );
+		test( 'should return an action object', () => {
+			const action = followList( 'restapitests', 'testlist' );
 
-			expect( spy ).toHaveBeenCalledWith( {
-				type: READER_LISTS_FOLLOW,
-				owner: 'restapitests',
-				slug: 'testlist',
+			expect( action ).toEqual( {
```

Comment:
What are we testing here? In what cases would the test fail other than if we change the structure of the action on purpose?